DNN Platform/Modules/ResourceManager/Dnn.Modules.ResourceManager.csproj

```diff
@@ -248,9 +254,16 @@
     <Content Include="App_LocalResources\EditFolderMapping.ascx.resx" />
   </ItemGroup>
   <ItemGroup>
-    <AdditionalFiles Include="..\..\..\stylecop.json">
-      <Link>stylecop.json</Link>
-    </AdditionalFiles>
+    <Analyzer Include="..\..\..\packages\StyleCop.Analyzers.1.1.118\analyzers\dotnet\cs\StyleCop.Analyzers.CodeFixes.dll" />
+    <Analyzer Include="..\..\..\packages\StyleCop.Analyzers.1.1.118\analyzers\dotnet\cs\StyleCop.Analyzers.dll" />
```

Comment:
These are already included on lines 269-270
```suggestion
```

Reply:
Another bad merge 🤦‍♂️. I'll accept the suggested change. Thanks!